.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Takich plików najlepiej nie commitować - widzę, że masz go w `.gitignore`, ale pewnie był już zastage'owany (przygotowany do zacommitowania) i dlatego nie został objęty.
https://stackoverflow.com/questions/13541615/how-to-remove-files-that-are-listed-in-the-gitignore-but-still-on-the-repositor